client/my-sites/jetpack-search/placeholder.tsx

```diff
@@ -35,17 +30,14 @@ export default function JetpackSearchPlaceholder( { siteId }: Props ): ReactElem
 			<DocumentHead title="Jetpack Search" />
 			<SidebarNavigation />
 
-			<FormattedHeader
-				headerText={ translate( 'Jetpack Search' ) }
-				id="jetpack-search-header"
-				align="left"
-				brandFont
+			<JetpackSearchContent
+				headerText={ translate( 'Finely-tuned search for your site.' ) }
+				bodyText={ translate( 'Your visitors are getting our fastest search experience.' ) }
```

Comment:
The bodyText, headerText and buttonText here don't seem to match?

Reply:
This is just dummy text and it's never shown to the user (hidden behind placeholder while loading). I'll make that clearer in the file.